Example/PrimerSDK/Info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSCameraUsageDescription</key>
+	<string>This app needs camera access to scan cards</string>
```

Comment:
Are we going to document the need for this permission for whoever will be integrating our SDK?

Reply:
Yup, it's only needed if `PrimerKlarnaSDK` is installed though